Engine/insert.c

```diff
@@ -1628,6 +1628,11 @@ int32_t setksmpsset(CSOUND *csound, SETKSMPS *p)
   return OK;
 }
 
+/* from udo.c */
+int32_t set_inbufs(CSOUND *csound,
```

Comment:
Seems like this should be moved to udo.h

Reply:
I think this was already done in develop so OK after merge.